### PR TITLE
Improved gateway.disconnect usage, Improved comments

### PR DESCRIPTION
**Pull request by sijocherian**
- Added gateway.disconnect() to a finally block
- gateway.disconnect() is not an async function
- Improve submitTransaction & evaluateTransaction comments

### CODE REVIEW

File: fabcar/javascript/invoke.js [hunk 1]
```diff
@@ -31,22 +31,26 @@ async function main() {
...
+
+            // Get the network (channel) our contract is deployed to.
+            const network = await gateway.getNetwork('mychannel');
+
+            // Get the contract from the network.
+            const contract = network.getContract('fabcar');
+
+            // Submit the specified transaction: The transaction function name will be evaluated on the
+            // endorsing peers and then submitted to the ordering service for committing to the ledger. 
+
```
Comment (btl5037) [inline fabcar/javascript/invoke.js:45]: There are stray tabs on line 45 that the linter has issues with

File: fabcar/javascript/query.js [hunk 2]
```diff
@@ -32,23 +32,27 @@ async function main() {
...
+        try {
+            await gateway.connect(ccp, { wallet, identity: 'appUser', discovery: { enabled: true, asLocalhost: true } });
+
+            // Get the network (channel) our contract is deployed to.
+            const network = await gateway.getNetwork('mychannel');
+
+            // Get the contract from the network.
+            const contract = network.getContract('fabcar');
+
+            // Evaluate the specified transaction. This is used for querying the world state.
```
Comment (btl5037) [inline fabcar/javascript/query.js:44]: Trailing spaces

File: fabcar/javascript/query.js [hunk 3]
```diff
@@ -32,23 +32,27 @@ async function main() {
...
+            // Get the network (channel) our contract is deployed to.
+            const network = await gateway.getNetwork('mychannel');
+
+            // Get the contract from the network.
+            const contract = network.getContract('fabcar');
+
+            // Evaluate the specified transaction. This is used for querying the world state. 
+            // The transaction function name will be evaluated on the endorsing peers but the responses
+            // will not be sent to the ordering service and hence will not be committed to the ledger. 
+
```
Comment (btl5037) [inline fabcar/javascript/query.js:47]: Stray tabs

**Comment by sijocherian** [comment]
Fixed lint issue. Updated typescript app with same.

**Pull request closed without merging**